base/src/main/java/uk/ac/ebi/atlas/search/suggester/SuggesterDao.java

```diff
@@ -56,7 +56,7 @@ public SuggesterDao(BioentitiesSolrClient solrClient) {
                 .setParam("suggest.dictionary", suggesterDictionary)
                 .setParam("suggest.q", query)
                 // We raise suggest.count to a high enough value to get exact matches (the default is 100)
-                .setParam("suggest.count", "500")
+                .setParam("suggest.count", "750")
```

Comment:
I don't quite understand this. I would assume (naïvely, perhaps) that the suggester prioritises exact matches... So no matter how low this count is, you'd still get some.

Reply:
Unfortunately, [the suggester implementation we use](https://lucene.apache.org/solr/guide/7_1/suggester.html#analyzinginfixlookupfactory) analyzes and tokenizes the input and performs matches against a Lucene index (which is very similar to a regular Solr query), so that’s not always the case.

There are [other](https://lucene.apache.org/solr/guide/7_1/suggester.html#analyzinginfixlookupfactory) [suggesters](https://lucene.apache.org/solr/guide/7_1/suggester.html#analyzinginfixlookupfactory) which include the boolean parameter `exactMatchFirst` but have no context filtering, which we need to filter suggestions by species. So, in principle, we could have one suggester per species, which I considered and maybe it’s better in the long term for the sake of correctness and performance. I think that for the time being this was the path of least resistance.

The 750 value was a bit of trial error, by the way...